airflow/migrations/versions/c381b21cb7e4_add_session_table_to_db.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add session table to db
+
+Revision ID: c381b21cb7e4
+Revises: be2bfac3da23
+Create Date: 2022-01-25 13:56:35.069429
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'c381b21cb7e4'
+down_revision = 'be2bfac3da23'
+branch_labels = None
+depends_on = None
+
+TABLE_NAME = 'session'
+
+
+def upgrade():
+    """Apply add session table to db"""
+    op.create_table(
```

Comment:
I think we should also add an ORM model for this table so that it's not only through migration that the db can be created.
I'm working on a change that would have new DBs created through the ORM rather than going through the migration files. https://github.com/apache/airflow/pull/21462

Reply:
We can do it like this I think

```python
from flask.session import SqlAlchemySessionInterface
from flask_appbuilder import SQLA
...

    db = SQLA()
    db.session = settings.Session
    SessionModel = SqlAlchemySessionInterface(db=db).sql_session_model
```

Reply:
(If we should or not is the next question)

Reply:
This is what it ends up looking like, when considering #21462's changes:

```
--- a/airflow/utils/db.py
+++ b/airflow/utils/db.py
@@ -606,12 +606,21 @@ def initdb(session: Session = NEW_SESSION):
     from airflow.models.base import Base
     from airflow.www.app import create_app
 
+    from airflow.www.session import AirflowDatabaseSessionInterface
+    from flask_sqlalchemy import SQLAlchemy
+
     Base.metadata.create_all(settings.engine)
     # Stamp migration head with alembic
     config = _get_alembic_config()
     command.stamp(config, "head")
     # sets up the db; Sync permissions etc
-    create_app(config={'UPDATE_FAB_PERMS': True})
+    app = create_app(config={'UPDATE_FAB_PERMS': True})
+
+    # Create `session` table
+    db = SQLAlchemy(app)
+    AirflowDatabaseSessionInterface(app=app, db=db, table='session', key_prefix='')
+    db.create_all()
+
     if conf.getboolean('core', 'LOAD_DEFAULT_CONNECTIONS'):
         create_default_connections(session=session)
 

```